CHANGES.rst

```diff
@@ -0,0 +1,14 @@
+Changelog
+=========
+
+Unreleased
```

Comment:
done

On 16 March 2015 at 17:13, Patrick <notifications@github.com> wrote:

> In CHANGES.rst
> <https://github.com/pnpnpn/timeout-decorator/pull/16#discussion_r26499855>
> :
>
> > @@ -0,0 +1,14 @@
> > +Changelog
> > +=========
> > +
> > +Unreleased
>
> We can do a release as 0.3.0. I set it up so that Travis will deploy to
> pypi when a PR is merged into master.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pnpnpn/timeout-decorator/pull/16/files#r26499855>.
>



-- 
Anatoly Bubenkov